datasets/simple-split/metadata.json

```diff
@@ -0,0 +1,58 @@
+{
+    "@context": {
+        "@vocab": "https://schema.org/",
+        "sc": "https://schema.org/",
+        "ml": "http://mlcommons.org/schema/",
+        "recordSet": "ml:RecordSet",
+        "field": "ml:Field",
+        "dataType": "ml:dataType",
+        "source": "ml:source"
+    },
+    "@type": "sc:Dataset",
+    "@language": "en",
+    "name": "simple-split",
+    "description": "An artificial example dataset defining splits from a CSV column",
+    "url": "https://mlcommons.org",
+    "license": "https://creativecommons.org/licenses/by/4.0/",
+    "distribution": [
+        {
+            "name": "books.csv",
+            "@type": "sc:FileObject",
+            "contentUrl": "data/books.csv",
+            "sha256": "",
```

Comment:
Did you omit the checksum on purpose?
This file's sha256 is d35c5a01eecbd7700faf86b4ec838eb65bd6e861633b1e10ca3294d4e58e75c9

Reply:
Thanks! I suppose it was just laziness, I am trying to use github codespace and haven't found a way to easily get the sha256 from there...